src/DataBase/EntryDatabase.cs

```diff
@@ -14,9 +15,11 @@ namespace ePiggyWeb.DataBase
     public class EntryDatabase
     {
         private PiggyDbContext Database { get; }
+        public string CnStr { get; set; }
         public EntryDatabase(PiggyDbContext database)
         {
             Database = database;
+            CnStr = "Server=51.75.187.147;Database=SmartSaver;User Id=usern;Password=123456789;";
```

Comment:
As I understand this is connection string, and I'm not sure but, we're saving it in app configs, so declaring it as string is a bad practice? @ciliakas could elaborate?

Reply:
Well I was thinking about the same thing, buuuut I think that since we're only using Data Adapter and Data Table for requirement purposes, and not actually what we would use in our app, I think this can slide. Anyway to be on the safe side we should ask the lecturer what he thinks